kolibri/plugins/learn/frontend/views/LibraryPage/index.vue

```diff
@@ -543,11 +542,10 @@
       },
       loading(newVal, oldVal) {
         if (oldVal && !newVal) {
-          const isTourStarted = this.resumeTour(this.userId, 'LibraryPage');
-          if (isTourStarted) {
+          if (!this.welcomeModalVisible) {
             setTimeout(() => {
               this.startTour('LibraryPage');
-            }, 3000);
+            }, 300);
```

Comment:
at this point, is this so it's not "abrupt" or is this to help manage some of the state issues that Shruti encountered earlier in development (just in a more narrow timeout window)?